bundles/org.eclipse.equinox.p2.engine/src/org/eclipse/equinox/p2/engine/ProvisioningContext.java

```diff
@@ -107,7 +107,15 @@ public boolean contains(IArtifactRepository element) {
 	 */
 	public static final String CHECK_AUTHORITIES = "org.eclipse.equinox.p2.director.checkAuthorities"; //$NON-NLS-1$
 
-	private static final String FOLLOW_ARTIFACT_REPOSITORY_REFERENCES = "org.eclipse.equinox.p2.director.followArtifactRepositoryReferences"; //$NON-NLS-1$
+	/**
+	 * Instructs the {@link PhaseSetFactory#PHASE_COLLECT collect phase} to check
```

Comment:
I would suggest to reuse the documentation from `FOLLOW_REPOSITORY_REFERENCES` and simply replace 'metadata repositories' with 'artifact repositories'.